contracts/Exchange.sol

```diff
@@ -679,20 +609,34 @@ contract Exchange is
         return (fundingGrowthGlobal, marketTwap, indexTwap);
     }
 
-    /// @dev get a price limit for replaySwap s.t. it can stop when reaching the limit to save gas
-    function _getSqrtPriceLimitForReplaySwap(address baseToken, bool isLong) internal view returns (uint160) {
+    /// @dev get a sqrt price limit for closing position s.t. it can stop when reaching the limit to save gas
+    function _getSqrtPriceLimitForClosingPosition(
+        address baseToken,
+        bool isLong,
+        uint160 inputSqrtPriceLimitX96
+    ) internal view returns (uint160) {
         int24 lastUpdatedTick = _lastUpdatedTickMap[baseToken];
         uint24 maxDeltaTick = _maxTickCrossedWithinBlockMap[baseToken];
 
-        // price limit = max tick + 1 or min tick - 1, depending on which direction
-        int24 tickBoundary =
-            isLong ? lastUpdatedTick + int24(maxDeltaTick) + 1 : lastUpdatedTick - int24(maxDeltaTick) - 1;
+        // price limit = max tick or min tick , depending on which direction
```

Comment:
max o min tick is a bit confusing. suggest to update to
```suggestion
        // price limit = upper tick boundary or lower tick boundary depending on which direction
```

Reply:
fixed in https://github.com/perpetual-protocol/perp-curie-contract/pull/90/commits/02d489f07729f89f036e0a7c0bd2eb884b6630ee